html/about.html

```diff
@@ -0,0 +1,161 @@
+    </section>
+    <hr />
+    <!-- TEDx logo -->
+    <section class="tedx-21-logo flex-h g-padding">
+      <h2 class="normal-title text-c">TEDx 2021 Logo</h2>
+      <div class="line"></div>
+      <p class="normal-desc text-c">
+        TED began in 1984 as a conference where Technology, Entertainment and
+        Design converged, and today covers almost all topics — from science to
+        business to global issues — in more than 100 languages. Meanwhile,
+        independently run TEDx events help share ideas in communities around the
+        world.
+      </p>
+      <img class="tedx-logo" src="../images/tedx-logo.jpg" alt="TEDx's logo" />
+    </section>
+    <hr />
+    <!-- Past TEDx -->
+    <section class="flex-h past-tedxs g-padding">
+      <h2 class="normal-title text-c">See the past TEDx events</h2>
+      <div class="line"></div>
+      <p class="normal-desc text-c">
+        Take a look at the last two TEDx events took place in different parts of
+        the world.
+      </p>
+      <div class="past-tedx-wrapper flex-h flex-h-d">
+        <div class="uni previous-event flex-h">
+          <h2 class="text-c previous-date">2020</h2>
+          <p class="text-c previous-place">TEDx Kabul University 2020</p>
+        </div>
+        <div class="kabul previous-event flex-h">
+          <h2 class="text-c previous-date">2019</h2>
+          <p class="text-c previous-place">TEDx Kabul Darulaman 2019</p>
+        </div>
+      </div>
+    </section>
+    <section class="partners flex-h">
+      <h2 class="partners-title normal-title">Partners</h2>
+      <div class="line"></div>
+      <div class="partners-logos">
+        <div class="partners-1">
+          <img
+            class="partner"
+            src="/images/partners/Skoll_logo.png"
+            alt=" logo"
+          />
+          <img
+            class="partner"
+            src="/images/partners/P_G_logo.png"
+            alt=" logo"
+          />
+          <img
+            class="partner"
+            src="/images/partners/ypf_logo.png"
+            alt=" logo"
+          />
+        </div>
+        <div class="partners-1">
+          <img
+            class="partner"
+            src="/images/partners/logitech_logo.png"
+            alt=" logo"
+          />
+          <img
+            class="partner"
+            src="/images/partners/JetBlueLogo.png"
+            alt=" logo"
+          />
+        </div>
+      </div>
+    </section>
+    <footer class="footer flex g-padding">
+      <img class="footer-logo footer-logo-desk" src="../images/tedx-logo.png" alt="TEDx's Logo" />
+      <img class="footer-logo footer-logo-mobile" src="../images/tedx-logo.jpg" alt="TEDx's Logo" />
+      <p class="copyright">
+        Copyright &copy;
+        <script>
+          document.write(new Date().getFullYear());
+        </script>
+        Wahidullah Shadab <br />
+        All Rights Reserved
+      </p>
+    </footer>
```

Comment:
- [ ] Hello. The footer on desktop view has a dark background and white background on mobile view but on your side on mobile view, there is no footer. Please fix that.
- [ ] The About page link is broken. Please also fix it.

Reply:
By the last the about page link, which link do you mean exactly. could you please specify that link?

Reply:
The footer looks fine here. I don't see any problem with it. It's with a dark background on desktop and with a white background on mobile. Can you please re-check it?
![image](https://user-images.githubusercontent.com/51750930/133690240-f41f0a60-f3b1-44a5-9006-547019c75530.png)
![image](https://user-images.githubusercontent.com/51750930/133690294-0633efe9-3176-4598-92b7-768453ba0556.png)